requirements-all.txt

```diff
@@ -1,6 +1,6 @@
 -r requirements.txt
 jsonargparse[signatures]  # CLI
-bitsandbytes==0.41.0      # quantization
+bitsandbytes>=0.41.0      # quantization
```

Comment:
BNB is not required by any other package.
How likely is that not the latest version can be installed? In other words, do we need `>=`?

Reply:
removed... and now it is `>`